tdtax/phenomenological.yaml

```diff
@@ -0,0 +1,34 @@
+class: Phenomenological
+tags: [Time series characteristics]
+subclasses:
+  - class: non-variable
+  - class: variable
+  - class: periodic
+    subclasses:
+      - class: sinusoidal
+        subclasses:
+          - class: ellipsoidal
+      - class: sawtooth
+  - class: multi periodic
+  - class: long timescale
+  - class: irregular
+  - class: eclipsing
+    subclasses:
+      - class: EA
+      - class: EB
+      - class: EW
```

Comment:
these seem to cross over into physical notions of eclipsing

Reply:
There is some crossover, although I think they can fit in the phenomenology because the light curve shapes of each type are separable into these categories without assuming the intrinsic source of their differences.

Reply:
Ok, makes sense.